configure

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
 
+import sys
+if sys.version_info >= (3, 0):
```

Comment:
Now that we are doing it, I like the version you proposed [here](https://github.com/nodejs/node/pull/9657#discussion_r88592166). Can we include that as well?

```python
if sys.version_info[0] != 2 or sys.version_info[1] not in (6, 7):
  sys.stdout.write("Please use either Python 2.6 or 2.7\n")
  sys.exit(1)
```